database/migrations/functions/users/get_user_tfa_config.sql

```diff
@@ -1,4 +1,4 @@
--- get_user_tfa_config returns the tfa configuration of the provided user.
+-- get_user_tfa_config returns the 2fa configuration of the provided user.
```

Comment:
(Ironically the commit message says tfa -- this happened because I made about a dozen tiny commits for individual 2fa changes and then instead of letting my tooling collapse, I tried to do it manually and ... oh well ... -- it looks like your commit strategy is squash-merge, so it should go out in the wash)

Reply:
That's all right, don't worry 👍